scripts/gatsby-plugin-checker/index.js

```diff
@@ -108,6 +109,31 @@ const updatePlugins = (updates, plugins) => {
   return updates
 }
 
+const filterArchived = plugins => {
+  const promises = plugins.map(plugin => {
+    const [username, packageName] = plugin.links.repository.split(`/`).slice(-2)
+    const url = `https://api.github.com/repos/${username}/${packageName}`
+    return got(url, {
+      headers: {
+        Authorization: `token ${process.env.GITHUB_API_TOKEN}`,
```

Comment:
do you mind adding a helpfull message if we forget to add a GITHUB_API_TOKEN ^^?

Reply:
Of course. That's great idea, thank you. I will add this as soon as i get home :+1: